src/pybind/mgr/rook/rook_cluster.py

```diff
@@ -125,6 +125,27 @@ def get_discovered_devices(self, nodenames=None):
 
         return nodename_to_devices
 
+    def list_pods(self, service_type):
```

Comment:
`self.list_pods(service_type)` looks very similar to `self.describe_pods(service_type, service_id=None)`. 

It looks like `describe_pods` cannot handle `service_type=None`. Can we make use of `describe_pods` here?